Core/Core/Common/CommonModels/AppEnvironment/BrandStyle/CDBrandVariables.swift

```diff
@@ -25,18 +25,18 @@ public final class CDBrandVariables: NSManagedObject {
     @NSManaged public var headerImageRaw: Data?
     @NSManaged public var institutionLogo: URL?
 
-    public private(set) lazy var brandVariables: APIBrandVariables? = {
+    public var brandVariables: APIBrandVariables? {
         let decoder = JSONDecoder()
         guard let jsonData = apiBrandVariablesRaw.data(using: .utf8) else {
             return nil
         }
         return try? decoder.decode(APIBrandVariables.self, from: jsonData)
-    }()
+    }
 
-    public private(set) lazy var headerImage: UIImage? = {
+    public var headerImage: UIImage? {
         guard let headerImageRaw else { return nil }
         return UIImage(data: headerImageRaw)
-    }()
+    }
```

Comment:
Why were the lazy properties changed to computed vars? Decoding and image creation will run each time someone accesses this property.